src/Services/OrderService/OrderService.Application/OrderList/CreateList/CreateListCommand.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Bookmarks.Domain;

```

Comment:
We should not use namespaces and classes from other services, as every single one should be independent and this creates a reliance and may lead to errors.